docs/amber_script.md

```diff
@@ -254,6 +254,16 @@ attachment content, depth/stencil content, uniform buffers, etc.
 
   # Bind the sampler at the given descriptor set and binding.
   BIND SAMPLER {sampler_name} DESCRIPTOR_SET _id_ BINDING _id_
+
+  # Bind OpenCL argument buffer by name. Specifying the buffer type is optional.
+  # Amber will set the type as appropriate for the argument buffer. All uses
+  # of the buffer must be across all pipelines.
+  BIND BUFFER {buffer_name} [AS {buffer_type}] KERNEL ARG_NAME _name_
+
+  # Bind OpenCL argument buffer by argument ordinal. Specifying the buffer type
+  # is optional. Amber will set the type as appropriate for the argument
+  # buffer. All uses of the buffer must be across all pipelines.
```

Comment:
Should say if it's 0-based.

Reply:
"all uses of the buffer must be ... ?consistent? ... across all pipelines"?

Do you mean: the inferred buffer type must be consistent across all pipelines?

(Because the descriptor set and binding can differ, it's important to say what aspect is consistent.)